src/components/developer-website/questionContainer/style.js

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+
+export const MainHeader = styled.div`
+  width: 38.813rem;
+  height: 23.376rem;
+  margin-left: 3.813rem;
+  margin-top: 6rem;
+
+  .header {
+    font-family: "Rubik";
+    margin-top: 0.1696rem;
+    color: #1a2534;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 3.75rem;
+
+    line-height: 4.438rem;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.035em;
+  }
+
+  .input {
+    font-family: "Rubik";
+    font-style: normal;
+    font-weight: 400;
+    width: 34.688rem;
+    font-size: 2.5rem;
+    line-height: 138.5%;
+    margin-top: 2rem;
+    border-top: 0rem;
+    border-right: 0rem;
+    border-left: 0rem;
```

Comment:
No need to include these if these are just zero anyway

Reply:
I added these because by not including these the style does not match the requirements.